pkg/cmd/init/options.go

```diff
@@ -12,6 +12,10 @@ type Options struct {
 	ConfigFlags *genericclioptions.ConfigFlags
 	factory     cmdutil.Factory
 	values      Values
+	//if set the resources will be sent to stdout instead of being applied
+	dryRun bool
```

Comment:
I think you can make it as a generic option instead of define it in every command.

Reply:
I will see what I can do to create genericOption for the clusteradm